### PR TITLE
Noviny/improvements

### PR DESCRIPTION
**Pull request by Noviny**
Realised this PR wasn't up yet. Three changes:

* No postgres, only sqlite, no 'db string' question
* Move auth into its own file, for easier code readability
* Add a readme to the generated content, so users have more pointers

### CODE REVIEW

File: create-keystone-app/starter/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Keystone Project Parter
+
+Welcome to Keystone!
+
+Run
+
+```
+npm keystone dev
```
Comment (bladey) [inline create-keystone-app/starter/README.md:8]: I think we could add `yarn dev` here as another option as we lead with it in the project README.md - https://github.com/keystonejs/create-keystone-app/blob/noviny/improvements/README.md

Perhaps:

---

To get started, run:

```yarn dev```

or

```npm run dev```

File: create-keystone-app/starter/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+We recommend you use this alongside our [getting started walkthrough](https://keystonejs.com/docs/walkthroughs/getting-started-with-create-keystone-app) which will walk you through what you get as part of this starter.
+
+If you want an overview of all the features keystone offers, check out our [why keystone](https://keystonejs.com/why-keystone) page.
+
+## Some Quick Notes On Getting Started
+
+### Changing the database
+
+We've set you up with an [sqlite database](https://keystonejs.com/docs/apis/config#sqlite) for ease-of-use. If you're wanting to use postgres, you can!
```
Comment (bladey) [inline create-keystone-app/starter/README.md:23]: Nitpicking, but there are a few references that we usually refer to in the right casing, example being `sqlite` usually stated as `SQLite` when not in configuration or code examples.

File: create-keystone-app/starter/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+For more on auth, check out our [authentication API docs](https://keystonejs.com/docs/apis/auth#authentication-api)
+
+### Adding a frontend
+
+Keystone is frontend agnostic, and provides a graphql endpoint you can write queries against at `/api/graphql` (by default [http://localhost:3000/api/graphql](http://localhost:3000/api/graphql)). At Thinkmill, we tend to use [Next.js](https://nextjs.org/) and [Apollo graphql](https://www.apollographql.com/docs/react/get-started/) as our frontend and way to write queries, but if you have your own favourite, feel free to pick that up.
+
+A walkthrough on how to do this is forthcoming, but in the meantime our [todo example](https://github.com/keystonejs/keystone-react-todo-demo) shows a keystone set up with a frontend. For a more full example, you can also look at an example app we built for [prisma day 2021](https://github.com/keystonejs/prisma-day-2021-workshop)
```
Comment (bladey) [inline create-keystone-app/starter/README.md:46]: Another example of casing, I feel it should be `Prisma Day 2021`.

File: create-keystone-app/starter/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+### Next embedded keystone
+
+While keystone works as a standalone app, you can embed your keystone app into a [Next.js](https://nextjs.org/) app. This is quite a different setup to the starter, and we recommend checking out our walkthrough for that [here](https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs#how-to-embed-keystone-sq-lite-in-a-next-js-app).
```
Comment (bladey) [inline create-keystone-app/starter/README.md:50]: `keystone` should be `Keystone`, I don't highlight them all though!

**Review by bladey — CHANGES_REQUESTED** [review]

**Comment by bladey** [comment]
Works really well, nice job.

File: create-keystone-app/starter/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+If you want an overview of all the features keystone offers, check out our [why keystone](https://keystonejs.com/why-keystone) page.
+
+## Some Quick Notes On Getting Started
+
+### Changing the database
+
+We've set you up with an [sqlite database](https://keystonejs.com/docs/apis/config#sqlite) for ease-of-use. If you're wanting to use postgres, you can!
+
+Just change the `db` property on line 16 of the keystone file [./keystone.ts](./keystone.ts) to
```
Comment (raveling) [inline create-keystone-app/starter/README.md]:
```suggestion
To use Postgres, just change the `db` property on line 16 of the keystone file [./keystone.ts](./keystone.ts) to
```

File: create-keystone-app/starter/README.md (hunk 2 above)
Comment (raveling) [reply create-keystone-app/starter/README.md]:
```suggestion
We've set you up with an [SQLlite database](https://keystonejs.com/docs/apis/config#sqlite) for ease-of-use.
```

File: create-keystone-app/starter/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    adapter: 'prisma_postgresql',
+    url: process.env.DATABASE_URL || 'DATABASE_URL_TO_REPLACE',
+}
+```
+
+For more on database configuration, check out or [db api docs](https://keystonejs.com/docs/apis/config#db)
+
+### Auth
+
+We have put auth into its own file to make this humble codebase easier to navigate. If you want to just browse without auth turned on, you can comment out the `isAccessAllowed` on line 21 of the keystone file [./keystone.ts](./keystone.ts) if you want to explore the setup and admin UI without worrying about auth right now.
```
Comment (raveling) [inline create-keystone-app/starter/README.md]:
```suggestion
We've put auth into its own file to make this humble starter easier to navigate. To explore it without auth turned on, comment out the `isAccessAllowed` on line 21 of the keystone file [./keystone.ts](./keystone.ts).
```

File: create-keystone-app/starter/README.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+For more on auth, check out our [authentication API docs](https://keystonejs.com/docs/apis/auth#authentication-api)
+
+### Adding a frontend
+
+Keystone is frontend agnostic, and provides a graphql endpoint you can write queries against at `/api/graphql` (by default [http://localhost:3000/api/graphql](http://localhost:3000/api/graphql)). At Thinkmill, we tend to use [Next.js](https://nextjs.org/) and [Apollo graphql](https://www.apollographql.com/docs/react/get-started/) as our frontend and way to write queries, but if you have your own favourite, feel free to pick that up.
```
Comment (raveling) [inline create-keystone-app/starter/README.md:44]:
```suggestion
As a Headless CMS, Keystone can be used with any frontend that uses GraphQL. It provides a GraphQL endpoint you can write queries against at `/api/graphql` (by default [http://localhost:3000/api/graphql](http://localhost:3000/api/graphql)). At Thinkmill, we tend to use [Next.js](https://nextjs.org/) and [Apollo graphql](https://www.apollographql.com/docs/react/get-started/) as our frontend and way to write queries, but if you have your own favourite, feel free to use it.
```

Comment (raveling) [reply create-keystone-app/starter/README.md:44]: Trying to posit the "Headless CMS" part here, as a means to help differentiate embedded mode below.

Comment (raveling) [reply create-keystone-app/starter/README.md:44]: It's an area many get confused with

- Keystone Next (good we got rid of that name)
- Admin UI build with Next.js
- Embedded mode, with Next.js

File: create-keystone-app/starter/README.md (hunk 4 above)
Comment (raveling) [reply create-keystone-app/starter/README.md]:
```suggestion
While Keystone is most commonly used in the "headless mode" mentioned above, you can also use it in "embedded mode" within a [Next.js](https://nextjs.org/) app. This isn't covered in this starter, but you can learn more about embedded mode [here](https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs#how-to-embed-keystone-sq-lite-in-a-next-js-app).
```

File: create-keystone-app/starter/README.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+A walkthrough on how to do this is forthcoming, but in the meantime our [todo example](https://github.com/keystonejs/keystone-react-todo-demo) shows a keystone set up with a frontend. For a more full example, you can also look at an example app we built for [prisma day 2021](https://github.com/keystonejs/prisma-day-2021-workshop)
+
+### Next embedded keystone
```
Comment (raveling) [inline create-keystone-app/starter/README.md:48]:
```suggestion
### Embedding Keystone in a Next.js frontend
```

File: create-keystone-app/starter/README.md (hunk 4 above)
Comment (raveling) [reply create-keystone-app/starter/README.md]: Not 100% sure I nailed this, but take a look at that article. Jed and I spent quite a bit of time labouring over the use of the term "modes" as the best way to frame out the embedded thing. Would be cool if this was consistent with that framing. 🙇

File: create-keystone-app/starter/README.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+
+```
+npm keystone dev
+```
+
+To view the config for your new app, look at [./keystone.ts](./keystone.ts)
+
+This project starter is designed to give you a sense of the power keystone can offer you, and show off some of its main features. It's also a pretty simple setup if you want to build out from it.
+
+We recommend you use this alongside our [getting started walkthrough](https://keystonejs.com/docs/walkthroughs/getting-started-with-create-keystone-app) which will walk you through what you get as part of this starter.
```
Comment (raveling) [inline create-keystone-app/starter/README.md]: Would this be better before the `npm run dev` bit ? 🤷‍♀️ 
(Not a hard opinion)

**Review by raveling — APPROVED** [review]

File: create-keystone-app/starter/README.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
+# Keystone Project Parter
```
[the remark this answers is not in the document]
Comment (Noviny) [reply create-keystone-app/starter/README.md:1]: Starter. I don't know how I got Parter.

**Review by bladey — APPROVED** [review]

**Pull request merged**